Georeferencing-Tin-12d-1/readme.md

```diff
@@ -0,0 +1,36 @@
+
+# UT-Georeferencing-Tin-12d-1 
+
+Tin in map coordinates
+
```

Comment:
```suggestion
```

Given the automatic procedure, it is not needed to have a level one heading in the readme.